uv.gyp

```diff
@@ -419,6 +419,11 @@
             'test/runner-unix.h',
           ],
         }],
+        [ 'OS in "mac dragonflybsd freebsd linux netbsd openbsd".split()', {
```

Comment:
`.split()` is more strict - doesn't allow substrings. Not a big deal in this situation.

```
>>> "o" in "foo bar"
True
>>> "o" in "foo bar".split()
False
```